src/lib/seam/components/DeviceDetails/NoiseSensorDeviceDetails.tsx

```diff
@@ -0,0 +1,149 @@
+                      for a specified time range.
+                    </span>
+                  </div>
+                ) : (
+                  'A noise threshold is the highest noise level (in dB) you want to allow for a given time range in the day.'
+                )
+              }
+            >
+              {!isInitialLoading && (noiseThresholds?.length ?? 0) > 0 ? (
+                noiseThresholds?.map((noiseThreshold) => (
+                  <DetailRow
+                    key={noiseThreshold.noise_threshold_id}
+                    label={
+                      <div className='seam-detail-row-label-column'>
+                        {noiseThreshold.name !== '' && (
+                          <span className='seam-detail-row-label'>
+                            {noiseThreshold.name}
+                          </span>
+                        )}
+                        <div className='seam-detail-row-label-block'>
+                          <span className='seam-row-sublabel seam-row-sublabel-text-default'>
+                            {formatTime(noiseThreshold.starts_daily_at)}
+                          </span>
+                          <ArrowRightIcon />
+                          <span className='seam-row-sublabel seam-row-sublabel-text-default'>
+                            {formatTime(noiseThreshold.ends_daily_at)}
+                          </span>
+                        </div>
+                      </div>
+                    }
+                  >
+                    <p>{noiseThreshold.noise_threshold_decibels} dB</p>
+                  </DetailRow>
+                ))
+              ) : (
+                <DetailRow
+                  label={
+                    <span className='seam-detail-row-empty-label'>None</span>
+                  }
+                />
+              )}
+            </DetailSection>
+
+            <div className='seam-detail-section-footer'>
+              <div className='seam-empty-div' />
+              <div className='seam-detail-section-footer-content'>
+                <div className='seam-detail-section-footer-content-text'>
+                  <p>{generateTimeZoneCaption(device, noiseThresholds)}</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </DetailSectionGroup>
+
+        <DeviceInfo
+          device={device}
+          disableConnectedAccountInformation={
+            disableConnectedAccountInformation
+          }
+          disableResourceIds={disableResourceIds}
+        />
+      </div>
+    </div>
+  )
+}
+
+const generateTimeZoneCaption = (
+  device: NoiseSensorDevice,
+  thresholds: NoiseThresholds[] | undefined
+): string | null => {
+  if (device.location?.timezone != null) {
+    return `All times in ${formatTimeZone(device.location.timezone)}`
+  }
+
+  if (thresholds == null) {
+    return null
+  }
+
+  const timeZone =
+    thresholds[0]?.starts_daily_at.match(/\[(.*?)\]/)?.[1] ?? 'UTC'
```

Comment:
What is this match for?

Reply:
Ahh, I will add a comment. It matches the timezone in the square brackets of the `starts_daily_at` string. Example: `'06:00:00[America/Los_Angeles]'`

Response value can be seen in the docs here: https://docs.seam.co/latest/api-clients/noise-sensors/list-noise-thresholds#list-noise-thresholds-set-on-device

Reply:
Oh yea, these are `seam-tod`. Can you try to parse it with this lib? I believe the API response should always be standardized to this: https://github.com/seamapi/zoned-time

Reply:
zoned-time is the successor to seam-tod, but the strings returned by the API should be compatible with zoned-time.

Reply:
Ahh, nice! Done.